surefire-api/src/test/java/org/apache/maven/surefire/api/testset/TestListResolverTest.java

```diff
@@ -401,19 +406,6 @@ public void testInclusiveWithDefaultExclusivePattern()
         assertTrue( runnable );
     }
 
-    public void testWildcard()
```

Comment:
This use case is still valid and the test should not be removed.

Reply:
but I removed ... `#optionallyWildcardFilter` so what I should check?

Reply:
@slawekjaranowski 
This test would be too simple if we have rewrite `optionallyWildcardFilter` to `**/*.class`.